src/main/java/pl/javastart/task/Car.java

```diff
@@ -0,0 +1,51 @@
+package pl.javastart.task;
+
+public class Car extends Vehicle {
+    private boolean airConditioningSystemStatus;
+    private static final double EXTRA_FUEL_CONSUMPTION_TOWARD_AIR_CONDITIONING = 0.8;
+
+    public Car(String name, double tankCapacity, double averageFuelConsumption, boolean airConditioningSystemStatus) {
+        super(name, tankCapacity, averageFuelConsumption);
+        this.airConditioningSystemStatus = airConditioningSystemStatus;
+    }
+
+    private String getAirConditioningSystemStatus() {
+        if (airConditioningSystemStatus) {
+            return "ON";
+        } else {
+            return "OFF";
+        }
+    }
+
+    public boolean isAirConditioningSystemStatus() {
+        return airConditioningSystemStatus;
+    }
+
+    public void turnAirConditionerOn() {
+        if (!airConditioningSystemStatus) {
+            this.airConditioningSystemStatus = true;
+        }
+    }
+
+    public void turnAirConditionerOff() {
+        if (!airConditioningSystemStatus) {
+            this.airConditioningSystemStatus = true;
+        }
```

Comment:
Ten `if` sprawdzający czy jest już włączone jest niepotrzebny. Nawet jeśli jest już wyłączone, to "zmiana" z false na false nie ma żadnych efektów ubocznych.

Poza tym, tutaj powinno być raczej `airConditioningSystemStatus = false;` (a nie `= true`)